src/vuestic-theme/vuestic-components/va-pagination/VaPagination.demo.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <VbDemo>
+    <VbContainer>
```

Comment:
This need will appear, because of the new suggested properties, that need to be described in demo with details. Table format was used before and is visual and easily perceived.
If you have strong argument against - ok, this representation will be changed.

Reply:
I don't have strong opinions in regards to demo format.